kbase-extension/static/kbase/js/widgets/narrative_core/catalog/app_card.js

```diff
@@ -1,341 +1,331 @@
+            this._renderAppCard = function() {
 
-            // Main Container
-            var $appDiv = $('<div>').addClass('kbcb-app-card kbcb-hover container');
+                var info = this.info;
+                var self = this;
 
-            // HEADER - contains logo, title, module link, authors
-            var $topDiv = $('<div>').addClass('row kbcb-app-card-header');
-            var $logoSpan = $('<div>').addClass('col-xs-3 kbcb-app-card-logo');
+                // Main Container
+                var $appDiv = $('<div>').addClass('kbcb-app-card kbcb-hover container');
 
-            if(type === 'method') {
-                $logoSpan.append('<div class="fa-stack fa-3x"><i class="fa fa-square fa-stack-2x method-icon"></i><i class="fa fa-inverse fa-stack-1x fa-cube"></i></div>')
-            } else if (type === 'app') {
-                $logoSpan.append('<span class="fa-stack fa-3x"><span class="fa fa-square fa-stack-2x app-icon"></span><span class="fa fa-inverse fa-stack-1x fa-cubes" style=""></span></span>');
-            }
+                // HEADER - contains logo, title, module link, authors
+                var $topDiv = $('<div>').addClass('row kbcb-app-card-header');
+                var $logoSpan = $('<div>').addClass('col-xs-2 kbcb-app-card-logo');
 
-            // add actual logos here
-            if(info.icon && nms_base_url) {
-                if(info.icon.url) {
-                    $logoSpan.html($('<img src="'+nms_base_url + info.icon.url+'">')
-                                        .css({'max-width':'85%', 'padding':'6px 3px 3px 8px',
-                                              'max-height': '85%'}));
+                if (this.type === 'method' || info.icon && info.icon.url && this.nms_base_url) {
+                    $logoSpan.append(Icon.makeAppIcon(this));
+                }
+                else if (this.type === 'app') {
+                    $logoSpan.append(Icon.makeGenericIcon('cubes'));
                 }
-            }
 
-            var $titleSpan = $('<div>').addClass('col-xs-9 kbcb-app-card-title-panel');
-                
-            $titleSpan.append($('<div>').addClass('kbcb-app-card-title').append(info.name));
-            if(info['module_name']) {
-                $titleSpan.append($('<div>').addClass('kbcb-app-card-module').append(
-                                        $('<a href="/#catalog/modules/'+info.module_name+'" target="_blank">')
-                                            .append(info.module_name)
-                                            .on('click',function(event) {
-                                                // have to stop propagation so we don't go to the app page first
-                                                event.stopPropagation();
-                                            })));
-            }
+                var $titleSpan = $('<div>').addClass('col-xs-10 kbcb-app-card-title-panel');
```

Comment:
Change this styling slightly.

The rest of the changes in this file are just whitespace reformatting.